### DISCUSSION TITLE
我尝试使用oidc组件是总是提示error status returned by host: bad argument

### DISCUSSION THREAD

**Discussion by Bearfirm** [higress-group/higress#800] [Q&A]
![image](https://github.com/alibaba/higress/assets/43485896/245e9fdb-433a-404c-8152-5517872a3b7f)
![image](https://github.com/alibaba/higress/assets/43485896/ee0e1408-3c8f-457d-93f5-06dd2ff1b9d4)

**Comment by Bearfirm** [comment]
我使用的是本地k8s部署
![image](https://github.com/alibaba/higress/assets/43485896/51389930-911e-45be-9e99-12a1e6a2e908)

**Comment by johnlanni** [comment]
cc @Fkbqf
